extension/manifest.json

```diff
@@ -23,7 +23,7 @@
   "browser_action": {
       "browser_style": true,
       "default_icon": {
-          "32": "icons/placeholder-logo-beta.png"
+          "32": "icons/make-new-alias.png"
```

Comment:
I've been mulling this over and I'm not sure. Since the primary functionality of the addon is to generate new email aliases, I thought it made sense to use the same icon that we're using to represent this functionality in the inputs. We could also modify up the "Generate New Relay Alias" button in the dashboard to use the icon as well, so that anytime you're generating a new alias, you're seeing the same icon. Ideally the logo and this icon would be nearly identical, so maybe once we have a high res version of the icon the brand team proposed, we can use that in all of the places and swap out the "Beta" for a "+" sign when the icon is used in-content to generate new aliases.